basic_commands/sections.md

```diff
@@ -23,28 +23,26 @@ Displaying information about sections:
 
 As you may know, binaries have sections and maps. The sections define the contents of a portion of the file that can be mapped in memory (or not). What is mapped is defined by the segments.
 
-Before the IO refactoring done by condret, the `S` command was used to manage what we now call maps. Currently the `S` command is supposed to be deprecated because `iS` and `om` should be enough.
+Before the IO refactoring done by condret, the `S` command was used to manage what we now call maps. Currently the `S` command is deprecated because `iS` and `om` should be enough.
 
-Firmware images, bootloaders and binary files usually place various sections of
-a binary at different addresses in memory. To represent this behavior, radare
-offers the `S` command. Use `S?` to get the help message. But we would recommend
-you to better check `om` and `iS` and try to avoid `S` as much as possible
-since it will be deprecated soon. To list all created sections use `S` (or `Sj` to get the json format). The `S=` will show the region bars in ascii-art.
+Firmware images, bootloaders and binary files usually place various sections of a binary at different addresses in memory. To represent this behavior, radare offers the `iS`. Use `iS?` to get the help message. To list all created sections use `iS` (or `iSj` to get the json format). The `iS=` will show the region bars in ascii-art.
 
-You can create a new section with a line like this:
+You can also use `om` command to view information about mapped sections:
 
 ```
-S [off] [va] [sz] [vsz] [name] [mrwx]
-```
-
-For example:
-
-```
-[0x00404888]> S 0x00000100 0x00400000 0x0001ae08 0001ae08 test rwx
-```
+[0x00401000]> om
+ 6 fd: 3 +0x00000600 0x00403000 - 0x004031ff -rwx fmap..what
+ 5 fd: 6 +0x00000000 0x00403200 - 0x00403fff -rwx mem..what
+ 4 fd: 3 +0x00000200 0x00401000 - 0x004011ff -rwx fmap..text
+ 3 fd: 5 +0x00000000 0x00401200 - 0x00401fff mrwx mmap..text
+ 2 fd: 3 +0x00000400 0x00402000 - 0x004021ff -rw- fmap..rdata
+ 1 fd: 4 +0x00000000 0x00402200 - 0x00402fff mrw- mmap..rdata
```

Comment:
Still need information on how to add a new map like it was previously

Reply:
I have added that part now.